melodic/distribution.yaml

```diff
@@ -8,6 +8,22 @@ release_platforms:
   ubuntu:
   - bionic
 repositories:
+  VisualStates:
+    doc:
+      type: git
+      url: https://github.com/JdeRobot/VisualStates.git
+      version: upstream/0.2.2
+    release:
+      packages:
+      - visualstates
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/JdeRobot/VisualStates.git
```

Comment:
This should be a different repository. Please see: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease#Creating_a_Release_Repository